data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -317,6 +318,8 @@
 
 REGISTERED_TIER_FITBIT_CLEANING_CLASSES = [
     (RemoveFitbitDataIfMaxAgeExceeded,),
+    (DeidFitbitDeviceId,
```

Comment:
Since this rule has to be run before the `FitbitPidRid` cleaning rule. we need to add `DeidFitbitDeviceId` in the depends_on list in the `FitbitPidRid` cleaning rule.